src/_site/index.html

```diff
@@ -10,6 +10,25 @@
         <link rel="stylesheet" type="text/css" href="./css/m.css" />
         <link rel="stylesheet" type="text/css" href="./css/default.css" />
         <link rel="shortcut icon" href="./images/favicon.ico" />
+        <style>
+            html, body {
+                height: 100%;
+            }
+            body {
+                height: 100%;
+            }
+            #wrapper {
+                min-height: 100%;
+            }
+            .main{
+                padding-bottom: 30px;
+                overflow: hidden;
+            }
+            footer {
+                padding-top: 0 !important;    /* Overwrite the useless padding */
+                margin-top: -70px !important; /* Equals to the height of footer element */
+            }
```

Comment:
It works if we also change the selector to ``footer.page-footer.blue-grey.darken-3``, but I think this looks ugly to add these classes which duplicate meaning.

I will change it if you wish.